graphics/GLC_lib/Portfile

```
@@ -37,7 +38,7 @@ configure.args      GLC_lib.pro
 
 # https://github.com/laumaya/GLC_lib/issues/30
 compiler.cxx_standard 2003
```

As previously discussed, C++03 is seemingly not required, and `compiler.cxx_standard` currently treats `2003` the same as `1998`. So let’s use:
```suggestion
compiler.cxx_standard 1998
```
for consistency with `-std=c++98`.